style.css

```diff
@@ -41,6 +52,40 @@ body {
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
+  margin: 3% auto;
+  margin-right: 10px;
+  margin-left: 10px;
+  margin-bottom: 20px;
+  border: 1px solid black;
+  width: 94%;
+  height: fit-content;
+  min-height: 94%;
+  border-radius: 12px;
+  padding: 16px;
+  zoom: 75%;
+}
+
+.see-pop {
+  display: flex;
+  justify-content: flex-start;
+  align-content: center;
+  flex-direction: row;
+  width: 90%;
+  gap: 20%;
+  padding-top: 20px;
+}
+
+.see-pop > .press {
+  min-width: 155px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-content: center;
+  background-color: #fff;
+  border: 1px solid #6070ff;

```

Comment:
- [ ] **Good job so far 👏 👏** However, as suggested by the previous reviewer, It would be better to make some adjustments to the spacing of your buttons to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=GdUM1FgslDNeTZT3-0) 🙏 🙏  

I have highlighted some areas to be improved ***Optional requirements are highlighted with yellow 😂***👇 
![image](https://user-images.githubusercontent.com/102757126/225547931-cd2798d1-ace0-4361-a19c-a903f80f2213.png)

- [ ] **[Optional]** Currently compared to the Figma design, your elements are small and it can't be easily readable, I would advise you to increase the size and the image also to make a good design 👍 👍 

_Replicating a Figma design in CSS is an important step in creating a well-designed, functional, and accessible website._